PE/__init__.py

```diff
@@ -719,7 +728,19 @@ def readPointerAtRva(self, rva):
         return self.readPointerAtOffset(off)
 
     def getMaxRva(self):
-        return self.IMAGE_NT_HEADERS.OptionalHeader.SizeOfImage
+        '''
+        Maximum RVA is the largest virtual address that might be observed.
+        '''
+        if not self.max_rva:
+            max_sec = 0
+            for sec in self.getSections():
+                sec_end = sec.VirtualAddress + sec.VirtualSize
+                align = self.IMAGE_NT_HEADERS.OptionalHeader.SectionAlignment
+                if (align > 0):
+                    sec_end = align * ((sec_end / align) + 1)
```

Comment:
Can you make the division a double slash (or wrap it in an int() call)? That way it stays an int when we move to py3.

Reply:
Done